README.md

```diff
@@ -24,3 +24,18 @@ Welcome to EpiCURE, a solution for restaurants struggling due to COVID-19!
   - Run `cd target/` to navigate to the directory with the `jar`
   - Run `java -jar remote-1.0-SNAPSHOT-jar-with-dependencies.jar "YOUR-PROJECT-ID"`
   - Ensure that "success" is printed!
+
+
+#### Google Cloud Scheduler
+In order to implement the scheduler needed to update a restaurant's score, you
+will need to complete the following steps:
+
+1. Go to your deployed Google Cloud Project's dashboard.
+2. Enter the naviation menu and loog for Cloud Scheduler under the Tools section.
+3. Select 'Create Job'.
+4. Complete all the fields needed to create the job.
+    a. Choose the name for the job.
+    b. Put the frequency as: "0 6 * * 7" (this means the job will run every Sunday at 6:00 AM).
+    c. Select your desired TimeZone.
+    d. The endpoint for the job is "/update-score" (that's what should go at the end of the rest of the URL for your site).T
+    e. he method should be a POST request and body should be empty.
```

Comment:
This list doesn't seem to be rendering correctly in the preview. 

If you go to the file in the pull request, in the right corner, just to the right of the " < >" button, there is a button to view the rich diff.